tests/commandline/test_create_init_perf_delete.csd

```diff
@@ -0,0 +1,34 @@
+<CsoundSynthesizer>
+<CsOptions>
+-n
+</CsOptions>
+<CsInstruments>
+0dbfs = 1
+
+opcode Osci(a:k,f:k):a
+xout linenr(oscili(p4,k(p5)),0.1,0.1,0.01) 
```

Comment:
Odd example that this opcode is taking in two params but not using them. Is the naming of a and f for arg names meant to test something in particular?

Reply:
done